src/components/organisms_new/CompareSyncPane/CompareModalSelecting/CompareModalSelecting.tsx

```diff
@@ -68,11 +69,15 @@ const CompareModalSelecting: React.FC = () => {
 
           <Col span={10}>
             {right.loading ? (
-              <Skeleton active />
+              <div style={{paddingLeft: hasSideSelected ? 6 : 0, height: '100%', overflow: 'hidden'}}>
+                <Loading />
+              </div>
             ) : right.error ? (
               <ErrorFigure onRetry={() => handleRetry('right')} />
             ) : (
-              <ResourceList data={right} />
+              <div style={{paddingLeft: 6}}>
```

Comment:
Can we add this padding directly in the `ResourceList` component? 🤔

Reply:
We cannot because it depends on the component's position. On the left hand, we don't need the padding, but on the right hand, we need to add it because of the scrollbar width